### DISCUSSION TITLE
用open-local-lvm存储类挂载容器某个目录，这个目录会被清空么

### DISCUSSION THREAD

**Discussion by xiayuxi** [alibaba/open-local#195] [Q&A] [answered]
用open-local创建了一个pvc01，  
 volumeMounts:
            - mountPath: /data
              name: mysql-server-claim0
      hostname: mysql-server
      restartPolicy: Always
      volumes:
        - name: mysql-server-claim0
          persistentVolumeClaim:
            claimName: pvc01
容器里挂载到data目录，/data目录之前存在的文件，被清空了。只有lost-found。

**Comment by peter-wangxu** [comment, marked as answer]
不建议这么用，PV所在的文件目录会overlay（遮挡）你容器层的文件夹内容。这里linux mount特性决定的

**Reply by xiayuxi** [reply]
好的，非常感谢，和我理解的一样。
